sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/eventhubs_stress_test.cpp

```diff
@@ -0,0 +1,296 @@
+namespace logs = opentelemetry::logs;
+namespace otlp = opentelemetry::exporter::otlp;
+namespace internal_log = opentelemetry::sdk::common::internal_log;
+
+void InitTracer()
+{
+  opentelemetry::exporter::otlp::OtlpHttpExporterOptions opts;
+
+  // Create OTLP exporter instance
+  auto exporter = otlp::OtlpHttpExporterFactory::Create(opts);
+  auto processor = trace_sdk::SimpleSpanProcessorFactory::Create(std::move(exporter));
+  std::shared_ptr<opentelemetry::trace::TracerProvider> provider
+      = trace_sdk::TracerProviderFactory::Create(std::move(processor));
+  // Set the global trace provider
+  trace::Provider::SetTracerProvider(provider);
+}
+
+// On debug builds, we log to the console. On release builds, we log to OpenTelemetry.
+#if defined(_DEBUG)
+constexpr const bool LogDefault = true;
+#else
+constexpr const bool LogDefault = false;
+#endif
+
+bool LogToConsole{LogDefault};
+
+void InitLogger()
+{
+  if (LogToConsole)
+  {
+    std::cout << "Using console to export log records." << std::endl;
+  }
+  else
+  {
+    opentelemetry::exporter::otlp::OtlpHttpLogRecordExporterOptions logger_opts;
+
+    std::cout << "Using " << logger_opts.url << " to export log records." << std::endl;
+    logger_opts.console_debug = false;
+
+    // Create OTLP exporter instance
+    auto exporter = otlp::OtlpHttpLogRecordExporterFactory::Create(logger_opts);
+    auto processor = logs_sdk::SimpleLogRecordProcessorFactory::Create(std::move(exporter));
+    std::shared_ptr<logs::LoggerProvider> provider
+        = logs_sdk::LoggerProviderFactory::Create(std::move(processor));
+
+    // Set the global log provider.
+    logs::Provider::SetLoggerProvider(provider);
+
+    // Integrate the azure logging diagnostics with the OpenTelemetry logger provider we just
+    // created.
+    Azure::Core::Diagnostics::Logger::SetListener(
+        [](Azure::Core::Diagnostics::Logger::Level level, std::string const& message) {
+          logs::Severity logSeverity{logs::Severity::kInvalid};
+          switch (level)
+          {
+            case Azure::Core::Diagnostics::Logger::Level::Error:
+              logSeverity = logs::Severity::kError;
+              break;
+            case Azure::Core::Diagnostics::Logger::Level::Informational:
+              logSeverity = logs::Severity::kInfo;
+              break;
+            case Azure::Core::Diagnostics::Logger::Level::Verbose:
+              logSeverity = logs::Severity::kDebug;
+              break;
+            case Azure::Core::Diagnostics::Logger::Level::Warning:
+              logSeverity = logs::Severity::kWarn;
+              break;
+          }
+          logs::Provider::GetLoggerProvider()
+              ->GetLogger(EventHubsLoggerName)
+              ->Log(logSeverity, message);
+        });
+
+    internal_log::GlobalLogHandler::SetLogLevel(internal_log::LogLevel::Error);
+  }
+}
+
+void CleanupTracer()
+{
+  // We call ForceFlush to prevent to cancel running exportings, It's optional.
```

Comment:
Comment is confusing

Reply:
I've improved the comment a bit, but it's literally C&P from the OpenTelemetry sample code.